pkg/systemd/logsJournal.jsx

```diff
@@ -137,7 +137,7 @@ export class JournalBox extends React.Component {
                 .done(() => {
                     this.setState({ streamFinished: true });
 
-                    if (!last) {
+                    if (!last && !promise.stopped) {
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19041-20230629-141509-f1fca553-fedora-38-devel-expensive//Coverage/lcov/github-pr.diff.gcov.html)